include/Page/Base.php

```diff
@@ -236,7 +237,7 @@ public function enqueue_script() {
 		$bulk_move_posts = array(
```

Comment:
All these are responsibility of the Custom Taxonomy metabox or the Ajax handlers.

Create a filter and then use the filter to add these values from inside those classes.